src/locales/en/general.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I agree `site.json` deals with client side components, and this is another case. However, we should consider adding these to `site.json` with a dedicated prefix (say `MARKDOWN_HOOK_`).

Otherwise, I suggest either renaming these new JSON files to `content.json` or anything that reflects their context, anything other than `general.json`.

Reply:
renamed